README.md

```diff
@@ -54,7 +54,7 @@ If you need help using or installing Node.js, please use the
 <http://nodeirc.info/> for more information regarding the `#node.js` IRC
 channel.
 
-_Please note that unofficial resources are neither managed by (nor necessarily
+_Please note that unofficial resources are neither managed by (not necessarily
```

Comment:
This change is simply incorrect (although removing the parentheses would be an improvement).